Apps/Sandcastle/gallery/development/Fog.html

```diff
@@ -0,0 +1,181 @@
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+<table><tbody>
+    <tr>
+        <td>enabled</td>
+        <td><input type="checkbox" data-bind="checked: enabled"/></td>
+    </tr>
+    <tr>
+        <td>start height</td>
+        <td><input type="text" size="5" data-bind="value: startHeight"></td>
+    </tr>
+    <tr>
+        <td>start density</td>
+        <td><input type="text" size="5" data-bind="value: startDensity"></td>
+    </tr>
+    <tr>
+        <td>end height</td>
+        <td><input type="text" size="5" data-bind="value: endHeight"></td>
+    </tr>
+    <tr>
+        <td>end density</td>
+        <td><input type="text" size="5" data-bind="value: endDensity"></td>
+    </tr>
+</tbody></table>
+<div id="zoomButtons"></div>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    "use strict";
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+viewer.terrainProvider = new Cesium.CesiumTerrainProvider({
+    url : '//assets.agi.com/stk-terrain/world',
+    requestWaterMask : true,
+    requestVertexNormals : true
+});
+
+viewer.extend(Cesium.viewerCesiumInspectorMixin);
+
+/*
+viewer.imageryLayers.removeAll();
+viewer.imageryLayers.addImageryProvider(new Cesium.TileMapServiceImageryProvider({
+    url : require.toUrl('Assets/Textures/NaturalEarthII')
+}));
+*/
+
+//viewer.scene.globe.enableLighting = true;
+
+//The viewModel tracks the state of our mini application.
+var viewModel = {
+    enabled : true,
+    startDensity : 0,
+    endDensity : 0,
+    startHeight : 0,
+    endHeight : 0
+};
+// Convert the viewModel members into knockout observables.
+Cesium.knockout.track(viewModel);
+
+// Bind the viewModel to the DOM elements of the UI that call for it.
+var toolbar = document.getElementById('toolbar');
+Cesium.knockout.applyBindings(viewModel, toolbar);
+
+Cesium.knockout.getObservable(viewModel, 'enabled').subscribe(
+    function(newValue) {
+        viewer.scene.fogEnabled = newValue;
+    }
+);
+    
+Cesium.knockout.getObservable(viewModel, 'startDensity').subscribe(
+    function(newValue) {
+        viewer.scene.fogStartDensity = newValue;
+    }
+);
+
+Cesium.knockout.getObservable(viewModel, 'endDensity').subscribe(
+    function(newValue) {
+        viewer.scene.fogEndDensity = newValue;
+    }
+);
+    
+Cesium.knockout.getObservable(viewModel, 'startHeight').subscribe(
+    function(newValue) {
+        viewer.scene.fogStartHeight = newValue;
+    }
+);
+
+Cesium.knockout.getObservable(viewModel, 'endHeight').subscribe(
+    function(newValue) {
+        viewer.scene.fogEndHeight = newValue;
+    }
+);
+
+viewModel.enabled = viewer.scene.fogEnabled;
+viewModel.startDensity = viewer.scene.fogStartDensity;
+viewModel.endDensity = viewer.scene.fogEndDensity;
+viewModel.startHeight = viewer.scene.fogStartHeight;
+viewModel.endHeight = viewer.scene.fogEndHeight;
+
+/*
+Sandcastle.addToolbarMenu([{
```

Comment:
Consider using the same views we used for OBBs: http://cesiumjs.org/2015/06/24/Oriented-Bounding-Boxes/

Instead of commenting them out, they could be in a drop down for future testing.